### DISCUSSION TITLE
How to get year, month, day and name from event?

### DISCUSSION THREAD

**Discussion by TheXer** [kuzmoyev/google-calendar-simple-api#73] [Q&A] [answered]
Hello, 
I'm trying to get the values from an Event object, is that in any way possible? What do I have now: 
```
for event in calendar:
        str(event)
        year, month, day, name = event.split("-")

        embed.add_field(name= f"{day}.{month}.{year}", value= name, inline=False)
```

Tuple doesn't work with the object and the code higher doesn't work too. Is it possible with different methods?

**Comment by kuzmoyev** [comment, marked as answer]
You can find all the fields of the event object [here](https://google-calendar-simple-api.readthedocs.io/en/latest/code/event.html).

In short:

```python
event.summary
event.start.year (.month/.day)
```


In general for python, try looking into `.__dict__` of the object:

    event.__dict__
